### ISSUE TITLE
Becoming a graduate should also happen through actionlink

### ISSUE THREAD

**Issue by rinkp** [GEWIS/gewisdb#295]
### What would you like?

Now that we have implemented graduate renewal, we can start thinking about the ordinary/external --> graduate process.

Once someone graduates, they should be given the option to become a graduate (and possibly, opt-out of Supremum, change their email address)

### Why is this needed?

Students can be registered using their TU/e email. If we don't email them within a certain timeframe, we are no longer able to do so

Moreover, the privacy aspect also plays a role: possibly students want their data removed once they graduate

**Comment by tomudding** [comment]
Before we start doing this we should make a clear diagram of all the different processes that determine what membership type someone has, how expiration is calculated, etc. Because changing this is going to make things a lot more complex.

While I do think that it is a good idea to send an e-mail to let the upcoming graduates know they may have to change their e-mail address I do not see the benefit of doing the actual conversion through an actionlink. As they should at all times be converted to graduate at the end of the association year, the form from the actionlink can be used to update information but should imho not be used for the conversion or extension of expiration.

**Comment by tomudding** [comment]
We also need to look at external -> external / graduate. The current implementation automatically renews an external member's membership if they still have the `$isStudying` flag.

**Comment by tomudding** [comment]
Folded into the Symfony migration. #564 makes `member.gewis.nl` a first-class host with its own routing, firewall and session, instead of an nginx rewrite over a loopback hop. Graduate renewal via actionlink is member self-service and belongs to that host controller set. Tracking epic: #555.
